rules/windows/process_creation/proc_creation_win_mstsc.yml

```diff
@@ -21,9 +22,9 @@ detection:
         - OriginalFileName: 'cmdkey.exe'
     selection_cmdkey_cli:
         CommandLine|contains|all:
-            - '/generic:'
-            - '/user:'
-            - '/pass:'
+            - '/g'
+            - '/u'
+            - '/p'
```

Comment:
Can't we put a space character before the slashes? 
Every character counts

Reply:
Yes We Can 🎶